async-http-client/src/test/scala/org/http4s/client/asynchttpclient/AsyncHttpClientSuite.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2016 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package client
+package asynchttpclient
+
+import cats.effect.{IO, Resource}
+import org.asynchttpclient.DefaultAsyncHttpClient
+import org.asynchttpclient.HostStats
+
+class AsyncHttpClientSpec extends ClientRouteTestBattery("AsyncHttpClient") with Http4sSuite {
+
+  def clientResource: Resource[IO, Client[IO]] = AsyncHttpClient.resource[IO]()
+
+  test(
+    "AsyncHttpClient configure should evaluate to the defaultConfiguration given the identity function as the configuration function") {
+    val defaultConfig = AsyncHttpClient.defaultConfig
+    val customConfig = AsyncHttpClient.configure(identity)
+
+    assertEquals(defaultConfig.getMaxConnectionsPerHost, customConfig.getMaxConnectionsPerHost)
+    assertEquals(defaultConfig.getMaxConnections, customConfig.getMaxConnections)
+    assertEquals(defaultConfig.getRequestTimeout, customConfig.getRequestTimeout)
```

Comment:
These don't need to be chained with `*>`?  It's just IO assertions that do?

Reply:
Exactly, IO assertions not chained won't be executed